src/datatypes/minecraft.js

```diff
@@ -25,35 +24,6 @@ function writeUUID(value, buffer, offset) {
   return offset + 16;
 }
 
-
-function readPosition(buffer, offset) {
-  var longVal = numeric.long[0](buffer, offset).value;
-  var x = signExtend26(longVal[0] >> 6);
-  var y = signExtend12(((longVal[0] & 0x3f) << 6) | ((longVal[1] >> 26) & 0x3f));
-  var z = signExtend26(longVal[1] & 0x3FFFFFF);
-  return {
-    value: {x: x, y: y, z: z},
-    size: 8
-  };
-}
-function signExtend26(value) {
```

Comment:
ah, nicely spotted, probably not. I'll need to add a signed: true option to the bitfields I guess